### ISSUE TITLE
Multiple comments per author returns with error

### ISSUE THREAD

**Issue by gvegayon** [r-lib/desc#65]
# What

I'm trying to add the ORCID component to the `comment` argument in the `person` function. While R CMD check works fine with comment fields that have more than one element per person (vectors), `desc::desc_normalize()` and `desc::add_author` fail when passing comments with more than one element:

```r
descr <- desc::desc(find.package("desc"))
descr$add_author(given = "Nice",
                 family = "Developer",
                 comment = c(ORCID = "orcid_number", what="he did it"))
descr$write("DESCRIPTION")
```

Returns with

```r
Error: comment must be a string or NULL
```
# Session info

```
Session info --------------------------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.4.4 (2018-03-15)
 system   x86_64, linux-gnu           
 ui       RStudio (1.1.419)           
 language en_US                       
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-05-24                  

Packages ------------------------------------------------------------------------------------------------------------------------------
 package    * version    date       source                             
 assertthat   0.2.0      2017-04-11 CRAN (R 3.4.0)                     
 backports    1.1.2      2017-12-13 cran (@1.1.2)                      
 base       * 3.4.4      2018-03-16 local                              
 clisymbols   1.2.0      2017-05-21 cran (@1.2.0)                      
 compiler     3.4.4      2018-03-16 local                              
 crayon       1.3.4      2017-09-16 CRAN (R 3.4.1)                     
 datasets   * 3.4.4      2018-03-16 local                              
 desc         1.2.0      2018-05-01 CRAN (R 3.4.4)                     
 devtools     1.13.5     2018-02-18 CRAN (R 3.4.3)                     
 digest       0.6.15     2018-01-28 cran (@0.6.15)                     
 graphics   * 3.4.4      2018-03-16 local                              
 grDevices  * 3.4.4      2018-03-16 local                              
 grid         3.4.4      2018-03-16 local                              
 highlight    0.4.7.2    2017-10-04 cran (@0.4.7.2)                    
 lattice      0.20-35    2017-03-25 CRAN (R 3.3.3)                     
 Matrix       1.2-14     2018-04-09 CRAN (R 3.4.4)                     
 memoise      1.1.0      2017-04-21 CRAN (R 3.4.0)                     
 methods    * 3.4.4      2018-03-16 local                              
 R6           2.2.2      2017-06-17 CRAN (R 3.4.0)                     
 rprojroot    1.3-2      2018-01-03 cran (@1.3-2)                      
 rstudioapi   0.7.0-9000 2018-05-24 Github (rstudio/rstudioapi@12870f8)
 stats      * 3.4.4      2018-03-16 local                              
 tools        3.4.4      2018-03-16 local                              
 usethis      1.3.0      2018-02-24 CRAN (R 3.4.3)                     
 utils      * 3.4.4      2018-03-16 local                              
 withr        2.1.2      2018-03-15 cran (@2.1.2)                      
 yaml         2.1.19     2018-05-01 CRAN (R 3.4.4)    
```

**Comment by maelle** [comment]
related to my own issue #63

**Comment by maelle** [comment]
@gvegayon this should have been fixed by #69, I ran your reprex again and it didn't produce any error. 😺

**Comment by gvegayon** [comment]
Works for me too! Thanks
